bin/src/middleware/validation/login.validation.ts

```diff
@@ -37,51 +38,68 @@ const verifyToken = async (
 };
 
 const checkLastlogin = async (user: JwtPayload) => {
-  const userRow = path.join(tableName, user.email);
-  const userDB: UserDto = await db.getObject<UserDto>(userRow);
+  try {
+    const user_entity = await UserEntity.findOne({
+      where: {
+        uuid: user.sub
+      }
+    })
+
+    if(!user_entity) {
+      return undefined;
+    }
 
-  const jti = user.jti;
-  const userJti = userDB.jti;
+    const jti = user.jti;
+    const userJti = user_entity.jti;
 
-  if (jti === userJti) {
-    const jwt_iat = user.iat?.toString();
-    const lastLogin = userDB.lastLogin;
+    if (jti === userJti) {
+      const jwt_iat = user.iat?.toString();
+      const lastLogin = user_entity.lastLogin;
 
-    if (jwt_iat !== lastLogin) {
-      throw new Error();
+      if (jwt_iat !== lastLogin) {
+        return undefined;
+      }
+
+      return true;
     }
-    return;
-  }
 
-  throw Error();
+    return undefined;
+  } catch (error) {
+    return handleErrorSync(error);
+  }
 };
 
 export const loginValidation = async (
   req: Request,
   res: Response,
   next: NextFunction,
 ) => {
-  const jwt_key = LcsConfig.get('jwtkey') || process.env.SECRET_KEY;
+  const jwt_key = (config.get('jwtkey') || process.env.SECRET_KEY)?.toString();
+
   if (!jwt_key) {
-    throw new Error(
+    throw new JsonWebTokenError(
       `No jwt_key for authentication provided. Run lcs config --jwtkey=<key>`,
     );
   }
 
   try {
-    const header = req.headers.authorization;
-    if (!header) {
+    const { authorization } = req.headers;
+
+    if (!authorization) {
       return res.status(401).json({ error: 'Unauthorized.' });
     }
 
-    const token = extractToken(header);
+    const token = await extractToken(authorization);
 
     req.user = await verifyToken(token, jwt_key);
 
-    await checkLastlogin(req.user);
+    const check = await checkLastlogin(req.user);
 
-    next();
+    if(check) {
```

Comment:
if `check` in not true, user won't get any error and stuck here. Throw errors in `checkLastLogin` instead of returning undefined. Refactor throwing and handling errors here in this file